app/src/main/java/com/example/android/codelabs/navigation/fragments/FlowStepFragment.kt

```diff
@@ -53,7 +52,7 @@ class FlowStepFragment : Fragment() {
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
 
-        next_button_step_one.setOnClickListener(
+        view.findViewById<View>(R.id.next_button).setOnClickListener(
```

Comment:
why you can't use synthetics here?

Reply:
i don't use synthetics here because this fragment works with layouts step_one_dest and step_two_dest too